src/rqt_msg/messages_widget.py

```diff
@@ -37,26 +37,29 @@
 from python_qt_binding.QtGui import QIcon
 from python_qt_binding.QtWidgets import (QAction, QMenu, QMessageBox,
                                          QTreeView, QWidget)
-import roslib
-import rosmsg
-import rospkg
-import rospy
 
+from rclpy import logging
 
-from .messages_tree_view import MessagesTreeView
-from rqt_py_common import rosaction
 from rqt_console.text_browse_dialog import TextBrowseDialog
 
+from rqt_msg.messages_tree_view import MessagesTreeView
 
-class MessagesWidget(QWidget):
+# from rqt_py_common import rosaction
+
+from rqt_py_common.topic_helpers import get_message_class, get_service_class
+from rqt_py_common.rqt_roscomm_util import iterate_packages
+from rqt_py_common import message_helpers
```

Comment:
These imports should be reversed and ordered alphabetically. Line 51, 50, then 49